chase/frontend/package.json

```diff
@@ -37,6 +37,7 @@
     "typescript": "5.0.4"
   },
   "devDependencies": {
+    "@types/js-cookie": "^3.0.3",
```

Comment:
what is this for?

Reply:
A skeleton from me trying out a few options on how to store the locale over a session